src/Mvc/test/Mvc.IntegrationTests/SimpleTypeModelBinderIntegrationTest.cs

```diff
@@ -699,6 +702,50 @@ public async Task BindParameter_FromFormData_BindsCorrectly(Dictionary<string, S
         Assert.Equal(new[] { "line 1", "line 2" }, entry.RawValue);
     }
 
+    [Fact]
+    public async Task BindParameter_PrefersTypeConverter_OverTryParse()
```

Comment:
Is it helpful to add a test that verifies the `TryParse` will get called for types that don't have a converter?